ee/debug/checkups/checkups.go

```diff
@@ -109,6 +109,7 @@ func checkupsFor(k types.Knapsack, target targetBits) []checkupInt {
 		{&desktopMenu{k: k}, flareSupported},
 		{&coredumpCheckup{}, doctorSupported | flareSupported},
 		{&downloadDirectory{}, flareSupported},
+		{&perfCheckup{}, doctorSupported | flareSupported | logSupported},
```

Comment:
I think we maybe either don't want this in doctor, or would only want a modified version in doctor? Since doctor is always run via the command line, the runtime stats will never be accurate, right?

Reply:
was thinking it would be nice in the doctor output in flares and didn't even consider that 🤦 you're absolutely right, will pull it